docs/DeveloperGuide.md

```diff
@@ -603,6 +606,101 @@ The following sequence diagram shows how the `login` command works:
 
 <puml src="diagrams/CustomerAddSequenceDiagram.puml" alt="CustomerAddSequenceDiagram" />
 
+### Customer Edit Command
+
+**Overview:**
+
+The `customer edit` command is used to edit an existing Customer with at least one of the information fields
+specified by the user, namely the customer's `Name`, `Phone`, `Email` or/and `Address`.
+
+The format for the `customer edit` command can be found [here](UserGuide.md#login).
```

Comment:
Should the link be linked to the customer edit feature in the UserGuide instead i.e., `UserGuide.md#edit`? Not sure about the exact reference.

Reply:
yep ok thanks